statsmodels/robust/norms.py

```diff
@@ -211,7 +211,7 @@ def rho(self, z):
                 (1 - test) * (np.fabs(z) * self.t - 0.5 * self.t**2))
 
     def psi(self, z):
-        """
+        r"""
```

Comment:
See http://www.statsmodels.org/stable/generated/statsmodels.robust.norms.HuberT.weights.html#statsmodels.robust.norms.HuberT.weights .  The old text renders correctly.  Usually only need r""" for latex.

Reply:
> The old text renders correctly. Usually only need r""" for latex

We'd have to ask past-me to be sure, but I think there were a bunch of warnings in the test output about invalid escapes.  These particular locations for r's would have been based on those warnings.

Reply:
Which format were the warnings for?  I don't think they are there for HTML.  Possibly LaTeX?